testing/btest/Baseline.cpp/language.count-underflow/.stderr

```diff
@@ -0,0 +1 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
```

Comment:
Presume there's no runtime warnings in the CPP version which the ZAM and interpreter versions have?

https://github.com/zeek/zeek/blob/ded011ef02a8876fa7b97d1f84b44954f776bddb/testing/btest/Baseline/language.count-underflow/.stderr#L1-L5

Reply:
Correct.  The thinking is that you use compile-to-C++ in production only after you're satisfied with the correctness of your script.  It still looks for run-time errors, though.